grafanalib/cloudwatch.py

```diff
@@ -32,6 +32,7 @@ class CloudwatchMetricsTarget(Target):
     :param hide: controls if given metric is displayed on visualization
     :param datasource: Grafana datasource name
```

Comment:
Please could you add a documentation string for this new parameter? 
```suggestion
    :param datasource: Grafana datasource name
    :param queryMode: adnafoinsdfinsoginsoigsfoigsfog
```

Reply:
I added  a comment